ssh/tunnel.py

```diff
@@ -81,6 +82,9 @@ def tunnel(user: str, key_path: str, host: str, port: int=22):
     logger.debug('Closing SSH Tunnel: ' + ' '.join(close_tunnel))
     check_call(close_tunnel)
 
+    if os.path.exists(tunnel_socket.name):
+        os.remove(tunnel_socket.name)
```

Comment:
Definitely agree to add a comment.

@branden Frankly, this file doenst even need to be deleted and I would rather completely remove this bit and leave it at NamedTemporaryFile(delete=False). Creating an entire context manager just to cover the case where we might have an early exception and then leave this harmless file seems like cruft in my opinion